Core.DomainModel/Organization/OrganizationUnit.cs

```diff
@@ -154,5 +154,49 @@ public bool IsUsed()
         {
             return Using.Any() || EconomyStreams.Any() || ResponsibleForItContracts.Any() || Rights.Any();
         }
+
+        public OrganizationRegistrationDetails GetUnitRegistrations()
+        {
+            return new OrganizationRegistrationDetails
+            (
+                Rights.ToList(),
+                ResponsibleForItContracts.ToList(),
+                GetUnitPayments().ToList(),
+                Using.Where(x => x.ResponsibleItSystemUsage != null).Select(x => x.ResponsibleItSystemUsage).ToList(),
+                Using.Select(x => x.ItSystemUsage).ToList()
+            );
+        }
+
+        public Result<OrganizationUnitRight, OperationError> GetRight(int rightId)
```

Comment:
change to Maybe<OrganizationUnitRight>... the result is already binary for an Id lookup